ui/pages/asset/components/__snapshots__/asset-page.test.tsx.snap

```diff
@@ -54,12 +54,12 @@ exports[`AssetPage should render a native asset 1`] = `
           </p>
         </button>
         <button
-          class="icon-button coin-overview__button icon-button--disabled"
+          class="icon-button w-16 icon-button--disabled opacity-30 cursor-auto coin-overview__button"
```

Comment:
Here we still retain any custom classname that is being added it's just been moved to the end